### ISSUE TITLE
Reference example for millis() incomplete

### ISSUE THREAD

**Issue by Veroaghe** [processing/p5.js#5485]
#### Most appropriate sub-area of p5.js?

- [ ] Accessibility (Web Accessibility)
- [ ] Build tools and processes
- [ ] Color
- [ ] Core/Environment/Rendering
- [ ] Data
- [ ] DOM
- [ ] Events
- [ ] Friendly error system
- [ ] Image
- [ ] IO (Input/Output)
- [ ] Localization
- [ ] Math
- [ ] Unit Testing
- [ ] Typography
- [ ] Utilities
- [ ] WebGL
- [x] Other (specify if possible):   
[example at "millis()" reference page on p5.js website is not updating.](https://p5js.org/reference/#/p5/millis)
The code isn't set within a draw() function.
It's my first time reporting an issue somewhere. Please, let me know if I'm doing something wrong.
Thank you.

#### Details about the bug:

- p5.js version: 
- Web browser and version: 
- Operating System: 
- Steps to reproduce this:

**Comment by reejuBhattacharya** [comment]
Hi!
Can you give some more details on what exact issue you are facing?

**Comment by Veroaghe** [comment]
[The example at "millis()" reference page on p5.js website is not updating.](https://p5js.org/reference/#/p5/millis)
The code isn't set within a draw() function, so the canvas is only showing the first frame. If it were set inside a draw() function, the number would continuously get larger.

**Comment by awelles** [comment]
Created a pull request for the website repo. 

https://github.com/processing/p5.js-website/pull/1100

Changes the example from: 

```js
let millisecond = millis();
text('Milliseconds \nrunning: \n' + millisecond, 5, 40);
```

To:

```js
function draw() {
  clear();
  let millisecond = millis();
  text('Milliseconds \nrunning: \n' + millisecond, 5, 40);
}
```

so the clock ticks.

**Comment by montoyamoraga** [comment]
closed due to explanation on [#1100](https://github.com/processing/p5.js-website/pull/1100)
